protocols/migration/migration_reports_3_0_0_to_participant_1_0_0.py

```diff
@@ -53,13 +54,27 @@ def migrate_tumor_sample(self, old_cancer_sample):
         new_tumour_sample.preparationMethod = old_cancer_sample.preservationMethod
         new_tumour_sample.source = participant_1_0_0.SampleSource.TUMOUR
 
+        new_tumour_type = None
+        if isinstance(old_cancer_sample.tumorType, str):
+            old_tumour_type = old_cancer_sample.tumorType.upper()
+            new_tumour_type = getattr(participant_1_0_0.TumourType, old_tumour_type, None)
+        new_tumour_sample.tumourType = new_tumour_type
+
+        new_tumour_content = None
+        if isinstance(old_cancer_sample.tumorContent, str):
```

Comment:
@priesgo can we please change this to:
```
if isinstance(old_cancer_sample.tumorContent, basestring):
```
@Antonior26 correctly pointed out that this will fail for unicode